### ISSUE TITLE
No such file or directory: 'prompts/rec_alignment.txt'

### ISSUE THREAD

**Issue by wzhwzhwzh0921** [zyang1580/CoLLM#13]
Hi there! I'm trying to train collm_sasrec version, but i can't find the prompt named rec_alignment.txt. Is it 'collm_movie.txt' or 'collm_amazon.txt'?

**Comment by zyang1580** [comment]
> Hi there! I'm trying to train collm_sasrec version, but i can't find the prompt named rec_alignment.txt. Is it 'collm_movie.txt' or 'collm_amazon.txt'?

Yes. To enhance the clarity of the files, I have changed their names. However, I didn't modify all the training configurations but just provided an example in the "Readme.md". Please refer to the "Training" section in "Readme.md" to set the prompt files.
